tests/auth_tests/test_views.py

```diff
@@ -878,6 +878,13 @@ def test_logout_preserve_language(self):
         self.client.get('/logout/')
         self.assertEqual(self.client.session[LANGUAGE_SESSION_KEY], 'pl')
 
+    @override_settings(LOGOUT_REDIRECT_URL='/custom/')
+    def test_logout_redirect_url_setting(self):
+        self.login()
+        response = self.client.get('/logout/')
+        self.assertEqual(response.status_code, 302)
```

Comment:
You could replace the two assertions with `self.assertRedirects(response, '/custom/', fetch_redirect_response=False)`